tests/lib/rules/space-unary-ops.js

```diff
@@ -418,6 +418,46 @@ ruleTester.run("space-unary-ops", rule, {
             }]
         },
         {
+            code: "+ +foo",
+            output: "+ +foo",
```

Comment:
As of 18a519fa2971e54a639e2b3320aeecd9a31b390a, we can now use `output: null` to assert that no autofixing occurs, so it would be better to do that here rather than setting the output to the code.